Rain-Drops/style.css

```diff
@@ -0,0 +1,619 @@
+* {
```

Comment:
Try to keep one format to writing styles.
if you decide to have space after each class name, need to do it every very.